### PR TITLE
Fix some bugs at README

### LINKED ISSUE #50: Add contributing information to README
As a team, we want to clean up our README, add contributing information and replace `yarn` with `npm`.
---

## :white_check_mark: Conditions of satisfaction
- [x] Should include contributing information
- [x] Update badges
- [x] Should replace `yarn` with `npm` (please check again)
- [x] It should be checked that a new setup is possible with the steps in the README.

### PR DESCRIPTION
**Pull request by lukashueller**
Fixes #50 

## Changes
* fix Heroku badge
* fix 2020 typo
* add Heroku hints to get invited to the project

## Screenshots
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # ark_automate
 
 [![GitHub stars](https://img.shields.io/github/stars/bptlab/ark_automate)](https://github.com/bptlab/ark_automate)
```
Comment (kej-jay) [inline README.md:3]: In my opinion we should only use badges for additional information not necessarily visible at first glance. Please think about what can be seen by scrolling up and what is additional information that might be of interest when on the main page of this projects repo

File: README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-![GitHub repo size](https://img.shields.io/github/repo-size/bptlab/ark_automate)
 [![GitHub open issues](https://img.shields.io/github/issues/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
 [![GitHub open pull requests](https://img.shields.io/github/issues-closed/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
 [![GitHub open pull requests](https://img.shields.io/github/issues-pr/bptlab/ark_automate)](https://github.com/bptlab/ark_automate/issues)
-
-<!-- [![heroku](heroku-badge.herokuapp.com/?app=ark-automate)](ark-automate.herokuapp.com) -->
+[![heroku](https://heroku-badge.herokuapp.com/?app=ark-automate&root=App.js)](https://heroku-badge.herokuapp.com/App.js)
 
 An open source RPA tool which uses BPMN to create bots.
```
Comment (WolfgangDaniel) [inline README.md:9]: I would rather describe it as "An open source RPA tool which uses several modeling languages to create bots."

**Review by WolfgangDaniel — CHANGES_REQUESTED** [review]
For the installation and setup and the configuration of the modeler I think it will be enough to just run "npm i" once in the client folder and once in the server folder because all the necessary dependencies are already in the package.json files. 
Moreover two parts of the readme (Set Up Heroku, Run Application) will be outdated once the refactoring of the folder structure is pushed. It should be an easy fix by  replacing "root directory" with "server directory" in these sections of the readme :) .

File: README.md (hunk 2 above)
Comment (lukashueller) [reply README.md]: I have adopted your text. Can you resolve the conversation? @WolfgangDaniel

**Comment by lukashueller** [comment]
> For the installation & setup and the configuration of the modeler I think it will be enough to just run "npm i" once in the client folder and once in the server folder because all the necessary dependencies are already in the package.json files.
> Moreover two parts of the readme (Set Up Heroku, Run Application) will be outdated once the refactoring of the folder structure is pushed. It should be an easy fix by replacing "root directory" with "server directory" in these sections of the readme :) .

Hi @WolfgangDaniel,
very good point! I've deleted some imports and and have put the checking of all other open imports into the issue "Clean up `package.json`". #57 

If you want, you can comment on @kej-jay remark. Then we should be able to merge the PR.

**Review by WolfgangDaniel — APPROVED** [review]

**Comment by WolfgangDaniel** [comment]
@lukashueller  in my opinion we can merge it now.

File: README.md (hunk 1 above)
Comment (lukashueller) [reply README.md:3]: Daniel said nothing about this. What do you think, @SanJSp?
I wouldn't mind the few badges. I think it's quite attractive that way.

**Pull request merged**